src/__swaps__/screens/Swap/hooks/useEstimatedGasFee.ts

```diff
@@ -47,7 +47,7 @@ export function useEstimatedGasFee({
     const feeFormatted = formatUnits(safeBigInt(fee), nativeNetworkAsset.decimals).toString();
     const feeInUserCurrency = multiply(networkAssetPrice, feeFormatted);
 
-    return convertAmountToNativeDisplayWorklet(feeInUserCurrency, nativeCurrency);
+    return convertAmountToNativeDisplayWorklet(feeInUserCurrency, nativeCurrency, true);
```

Comment:
this is for APP-1601: gas button can display a fee of $0.00 instead of proper handling of amounts below $0.01.

we had consolidated the `convertAmountToNativeDisplayWorklet` and the `convertAmountToNativeDisplayWithThresholdWorklet` so we should confirm that we correctly pass the `useThreshold` flag in the places we call this function

Reply:
PoW:
![Simulator Screenshot - iPhone 15 Pro - 2024-06-20 at 08 30 24](https://github.com/rainbow-me/rainbow/assets/1285228/e134f899-bb07-4dd4-94aa-e6571073079a)